admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+[[admin-guide-sched-pod-affinity]]
+= Using Pod Affinity and Anti-Affinity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+_Pod affinity_ and _pod anti-affinity_ allow you to specify rules about how pods should be placed relative to one another. For example, you could spread or pack pods within a service or relative to pods in other services or, you can make sure a pod is scheduled on a specific type of machine.
```

Comment:
@wmengRH This statement came from the Kubernetes doc: 
"there are some circumstances where you may want more control on a node where a pod lands, e.g. to ensure that a pod ends up on a machine with an SSD attached to it, or to co-locate pods from two different services that communicate a lot into the same availability zone."
I made the "machine with an SSD attached" to a more general statement. 
Maybe I can say instead: "you can make sure a pod is scheduled on **a specific machine**."
Would that make more sense?